SPECS/dwarves/dwarves.spec

```diff
@@ -117,7 +115,6 @@ rm -Rf %{buildroot}
 %doc MANIFEST README
 %{_includedir}/dwarves/btf_encoder.h
 %{_includedir}/dwarves/config.h
-%{_includedir}/dwarves/ctf_encoder.h
```

Comment:
This should be ok for 3.0 but definitely not for 2.0 (since it would break existing users who might be doing e.g. `#include <dwarves/ctf_encoder.h>`.

Also, removing header files usually isn't done very often, were all these header files actually removed? Might be good to at least double-check that...if they were removed upstream, that's fine, just would be good to take a sec to verify.

Also note this may break other packages we have that depend on this package (which is ok, assuming it's fixed when we update those packages, assuming there are any that break)

Reply:
Yup! I checked because it also seemed fishy. Arnaldo Carvalho de Melo is the upstream maintainer and is from redhat. Here is his commit https://src.fedoraproject.org/rpms/dwarves/c/550742fafde6bd74864491d12410404529ba7d4f?branch=rawhide

Reply:
Also this is only for 3.0 to match latest binutils :)